eventsourcing/src/test/java/org/axonframework/eventsourcing/eventstore/SimpleEventStoreTest.java

```diff
@@ -82,68 +82,6 @@ private static StreamingCondition aStreamingCondition() {
         return StreamingCondition.startingFrom(new GlobalSequenceTrackingToken(999));
     }
 
-    @Nested
-    class DelegatingToStorageEngine {
```

Comment:
I think these tests should move to the `StreamableEventSource` implementation instead of being removed entirely.

Reply:
For sure!